.circleci/config.yml

```
@@ -904,7 +904,7 @@ jobs:
           patterns: indexer
       - run:
           name: Lint
-          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint --timeout 2m -e "errors.As" -e "errors.Is" ./...
+          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint -e "errors.As" -e "errors.Is" ./...
```

sorry, I'm stupid, the golangci-lint default timeout is 1m (not 10m like golang test). We should probably bump it to like 4m

---

```suggestion
          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint - --timeout 4m -e "errors.As" -e "errors.Is" ./...
```

---

```suggestion
          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint --timeout 2m -e "errors.As" -e "errors.Is" ./...
```
@tynes is correct I force pushed away this change